glossary/public-member-definition.md

```diff
@@ -0,0 +1,9 @@
+---
+type: Resource
+---
+
+# Public Member
+
+A public member is a publicly owned organization that is officially part of, and able to govern, the Foundation for Public Code vereniging. They abide by our articles of association, pay membership dues, and help shape our processes by partaking in the governance and finance of our organization.
```

Comment:
I've suggested a less obscure word, which will make this easier for non-native English speakers.

```suggestion
A public member is a publicly owned organization that is officially part of, and able to govern, the Foundation for Public Code Vereniging. They abide by our articles of association, pay membership dues, and help shape our processes by participating the governance and finance of our organization.
```